draft-ietf-ppm-dap.md

```diff
@@ -1996,10 +2005,11 @@ struct {
 } PrepareResp;
 ~~~
 
-Next the Helper decrypts each of its remaining report shares as described in
+The helper processes each of the remaining report shares in turn:
```

Comment:
We have been capitalizing Client, Leader, Helper and Collector when referring to DAP roles.

```suggestion
The Helper processes each of the remaining report shares in turn:
```